### PR TITLE
MINOR: Fix quickstart in docs

### PR DESCRIPTION
**Pull request by vahidhashemian**
Reverting some of the recent changes to quickstart doc. Further explanation is provided inline.

### CODE REVIEW

File: docs/quickstart.html [hunk 1]
```diff
@@ -359,7 +359,7 @@
 
 
 <pre>
-&gt; <b>cat file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
```
Comment (vahidhashemian) [inline docs/quickstart.html:362]: `.sh` is missing.

File: docs/quickstart.html [hunk 2]
```diff
@@ -359,7 +359,7 @@
...
+&gt; <b>bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt</b>
```
Comment (vahidhashemian) [inline docs/quickstart.html:362]: This replacement had been made before to make the command work on both Linux and Windows systems. `cat` does not work on Windows.

File: docs/quickstart.html [hunk 3]
```diff
@@ -397,12 +397,9 @@
...
 all     1
-streams 1
 lead    1
 to      1
-kafka   1
 hello   1
-kafka   2
 streams 2
 join    1
 kafka   3
```
Comment (vahidhashemian) [inline docs/quickstart.html:405]: These removed lines do not appear when going through the exercise.

**Comment by vahidhashemian** [comment]
@guozhangwang @ewencp I ran into a couple of issues when going through the streams quickstart for 0.10.2.0-RC1 (http://kafka.apache.org/0102/documentation.html#quickstart_kafkastreams). I had addressed these issues a while back, but looks like those changes are reverted by a recent PR.

Please see further explanation inline. Your feedback is appreciated.

**Comment by ewencp** [comment]
@vahidhashemian Which PR reverted it? It's possible it is accidental -- we have been trying to get the apache/kafka-site and apache/kafka repos back in sync.

**Comment by vahidhashemian** [comment]
@ewencp It seems to be [this one](https://github.com/apache/kafka/pull/2482).

**Comment by ewencp** [comment]
@vahidhashemian Thanks, that's what I thought -- that was the PR trying to bring things back inline.

I'm going to merge this to trunk and 0.10.2. Ideally this shouldn't hold up the release since we can easily regenerate and redeploy the 0.10.2 docs after the release (since the are always just the latest version available on that release branch).

**Comment by guozhangwang** [comment]
I remember merging them on kafka-site repo but obviously copy-paste overrides them. We should consider having a nightly job for making the current trunk's web doc visible as well, like `0.10.3.0-SNAPSHOT/documentation` so that people only need to touch on kafka repo.

Thanks @vahidhashemian for the quick find and @ewencp for the quick response!

**Comment by ewencp** [comment]
@guozhangwang The alternative, of course, would be to make it so viewing the current copy of the docs didn't require so much setup... I've given in, but requiring having Apache specifically installed and not just supporting running a web server (e.g. python's SimpleHTTPServer that is available pretty much everywhere out of the box), is incredibly annoying. It means pretty much nobody except committers that are going to updates kafka-site ever properly test whether docs will render ok.

**Comment by guozhangwang** [comment]
As AK web docs gets fancier (we even have discussed about moving it from html docs to rst), I think such complexity would be hard to avoid. Having nightly builds for trunk may actually help with this, such that as long as people can accept some commits causing the web doc / links to be broken the next day, such commits can be fixed forward / reverted back in time, instead of piled only at the release time.

**Pull request closed without merging**